app/controllers/editions_controller.rb

```diff
@@ -116,6 +119,20 @@ def update_tagging
     }
   end
 
+  def update_related_external_links
+    artefact = resource.artefact
+    external_links = params.require(:artefact).permit(external_links_attributes: [:title, :url, :id, :_destroy])
+
+    artefact.external_links_attributes = external_links["external_links_attributes"]
+
+    if artefact.save
+      flash[:success] = "External links have been saved."
+    else
+      flash[:danger] = "Not saved. Links should begin with 'http'."
```

Comment:
I decided to return the original messages and stop testing for their content as the error messages body should only be tested on the model in govuk_content_models.